### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
Updated readme.md to include keyfeatures of ARES Dashboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,111 @@
...
-### Available Sample Code in 
+> ARES is a web based portal, which provides automation results at one place. It can integrate with most popular automation tools like Selenium, Protractor etc. This repo consist some samples and usage of ARES test automation dashboard
 
-* Java
-* Java Script
-* C#
-* Python (coming soon...)
+![HitCount](http://hits.dwyl.io/testastra/ares.svg)
+![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)
+![Open Source Love png1](https://badges.frapsoft.com/os/v1/open-source.png?v=103)
```
Comment (testastra) [inline README.md]: It is not open source but under GNU license

File: README.md [hunk 2]
```diff
@@ -1,11 +1,111 @@
...
+  - Test Automation Health Status
+  - Test Automation Execution Trends
+  - Top 5 failed test cases
+  - Test Execution Time Trends
+- Centralized report (view reports from any part of globe by login)
+
+---
+__Why ARES__
+
+- It's Open Source with unlimited features
```
Comment (testastra) [inline README.md]: Please make it "It is free" and remove "Unlimited features"

File: README.md [hunk 3]
```diff
@@ -1,11 +1,111 @@
...
+- No Setup or Installation required
+- Made by QA
+
+---
+__SAMPLE REPORT__
+
+ - Sample Report [link](https://www.youtube.com/watch?v=KiQDBw9kAHI&feature=youtu.be)
```
Comment (testastra) [inline README.md]: Right now, we just have only one youtube link for our webinar, and currently no sample reports available. we can publish few public and can add those references later here. For now, lets keep youtube reference only

File: README.md [hunk 4]
```diff
@@ -1,11 +1,111 @@
...
+In addition to your test automation code, you need to call 3 Restful APIs at designated places in your test automation framework. 
+
+These APIs would carry some information about your test automation such as number of tests to be executed, test name, test status, platform used, error logs etc to the dashboard server in JSON format, which would then represent the data in some graphs and charts. 
+
+---
+__CONFIGURATION / USAGE / TUTORIALS__
+
+List of available tutorials (comming soon...)
+
+ - Java [link1](https://www.google.com) | [link2](https://www.google.com)
```
Comment (testastra) [inline README.md]: replace each link with respective folder link in ARES github repo. eg. https://github.com/testastra/ARES/tree/master/Java

File: README.md [hunk 5]
```diff
@@ -1,11 +1,111 @@
...
+ - Javascript [link1](https://www.google.com) | [link2](https://www.google.com)
```
Comment (testastra) [inline README.md]: Just keep either protractor or javascript but not both.

**Review by testastra — COMMENTED** [review]
Suggested few textual and hyperlink changes for your review.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Removed open source badge

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Updated as suggested

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Included only one link

**Comment by ghost** [comment]
Made suggested changes. Please review

**Pull request merged**